TST: test_coordinate_operation_grids__alternative_grid_name failed

https://github.com/pyproj4/pyproj/actions/runs/3453353691/jobs/5763886591
```
def test_coordinate_operation_grids__alternative_grid_name():
        cc = CoordinateOperation.from_epsg(1312, True)
        assert len(cc.grids) == 1
        grid = cc.grids[0]
        assert grid.direct_download is True
        assert grid.open_license is True
        assert grid.short_name == "ca_nrc_ntv1_can.tif"
        if (PROJ_GTE_91 and grids_available(grid.short_name, check_network=False)) or (
            not PROJ_GTE_91 and grids_available(grid.short_name)
        ):
            assert grid.available is True
            assert grid.full_name.endswith(grid.short_name)
        elif PROJ_GTE_91 and pyproj.network.is_network_enabled():
            assert grid.available is True
>           assert grid.full_name == ""
E           AssertionError: assert 'https://cdn...._ntv1_can.tif/' == ''
E             + https://cdn.proj.org/ca_nrc_ntv1_can.tif
```

--- a/test/conftest.py
+++ b/test/conftest.py
@@ -13,6 +13,7 @@ PROJ_LOOSE_VERSION = version.parse(pyproj.__proj_version__)
 PROJ_GTE_9 = PROJ_LOOSE_VERSION >= version.parse("9.0.0")
 PROJ_GTE_901 = PROJ_LOOSE_VERSION >= version.parse("9.0.1")
 PROJ_GTE_91 = PROJ_LOOSE_VERSION >= version.parse("9.1")
+PROJ_GTE_911 = PROJ_LOOSE_VERSION >= version.parse("9.1.1")
 
 
 def unset_data_dir():

--- a/test/crs/test_crs.py
+++ b/test/crs/test_crs.py
@@ -20,7 +20,13 @@ from pyproj.crs.enums import CoordinateOperationType, DatumType
 from pyproj.enums import ProjVersion, WktVersion
 from pyproj.exceptions import CRSError
 from pyproj.transformer import TransformerGroup
-from test.conftest import PROJ_GTE_91, PROJ_GTE_901, assert_can_pickle, grids_available
+from test.conftest import (
+    PROJ_GTE_91,
+    PROJ_GTE_901,
+    PROJ_GTE_911,
+    assert_can_pickle,
+    grids_available,
+)
 
 
 class CustomCRS(object):
@@ -604,19 +610,22 @@ def test_coordinate_operation_grids__alternative_grid_name():
     assert grid.direct_download is True
     assert grid.open_license is True
     assert grid.short_name == "ca_nrc_ntv1_can.tif"
+    assert grid.package_name == ""
+    assert grid.url == "https://cdn.proj.org/ca_nrc_ntv1_can.tif"
     if (PROJ_GTE_91 and grids_available(grid.short_name, check_network=False)) or (
         not PROJ_GTE_91 and grids_available(grid.short_name)
     ):
         assert grid.available is True
         assert grid.full_name.endswith(grid.short_name)
+    elif PROJ_GTE_911 and pyproj.network.is_network_enabled():
+        assert grid.available is True
+        assert grid.full_name == grid.url
     elif PROJ_GTE_91 and pyproj.network.is_network_enabled():
         assert grid.available is True
         assert grid.full_name == ""
     else:
         assert grid.available is False
         assert grid.full_name == ""
-    assert grid.package_name == ""
-    assert grid.url == "https://cdn.proj.org/ca_nrc_ntv1_can.tif"
 
 
 def test_coordinate_operation__missing():

